docs/core/docker/introduction.md

```diff
@@ -31,7 +31,7 @@ Container registries are a collection of image repositories. You can base your i
 
 Docker has a public registry hosted at the [Docker Hub](https://hub.docker.com/) that you can use. [.NET Core related images](https://hub.docker.com/_/microsoft-dotnet-core/) are listed at the Docker Hub.
 
-The Microsoft Container Registry (MCR) is the official source of Microsoft-provided container images. The MCR is built on Azure CDN to provide globally-replicated images. However, the MCR does not have a public-facing website and the primary way to learn about Microsoft-provided container images is through the [Microsoft Docker Hub pages](https://hub.docker.com/_/microsoft-dotnet-core/).
+The [Microsoft Container Registry (MCR)](/azure/container-registry) is the official source of Microsoft-provided container images. The MCR is built on Azure CDN to provide globally-replicated images. However, the MCR does not have a public-facing website and the primary way to learn about Microsoft-provided container images is through the [Microsoft Docker Hub pages](https://hub.docker.com/_/microsoft-dotnet-core/).
```

Comment:
I feel like this may be confusing to readers. The link doesn't list images, it's just to the backing host for those images, but is presented in a way that is "buy this now for your own images" which isn't helping users get to the images for docker. I feel the link at the end, Microsoft Docker Hub pages, is the appropriate link.

Reply:
I agree with that, but I also think the link to the MCR service is meaningful as well. It's kind of a special thing, and users might be curious about it - no?

Reply:
I edited my response a few times, you may be responding to and older one 😁 

My final draft points out this isn't the MCR, the MCR doesn't have a link.